### ISSUE TITLE
FQDN is presented 2 times in file hyperlinks

### ISSUE THREAD

**Issue by artoonl** [atrocore/atrocore#1287]
Hi!,

After upgrading to 1.10.23 the assets are moved to files. 
Now in the UI the links to files (images and csv files) are broken, because the FQDN is presented 2 times see screenshot.
Can I fix this locally? 

![2x-fqdn-thumbnailpath](https://github.com/atrocore/atrocore/assets/16335222/2be15e28-8861-44d1-b008-b472a747db5e)
![Atrocore-doubleFQDN-filedownload](https://github.com/atrocore/atrocore/assets/16335222/2cd6539f-9879-46dd-a57a-7f3be718f640)
![Atrocore-doubleFQDN-filedownload2](https://github.com/atrocore/atrocore/assets/16335222/5ca5d3ed-9f8a-4aab-aa30-f452e00e138d)

**Comment by rratsun** [comment]
Hello, try to set **Site URL** in **Administration / Settings**

**Comment by rratsun** [comment]
Also try to clear cache in your browser and also in **Administration / Clear cache**

**Comment by artoonl** [comment]
Thnx rratsun! 

I tried to reset the site URL and cleared the cache but no result, the GUI still presents a double FQDN in file hyperlinks on showing the thumb and download button.

**Comment by rratsun** [comment]
Unfortunately, this should not be. You may have changed the code and it somehow affected it, or the update may not have been performed in the standard way. Maybe it's still the cache. The update could not have affected this, as we would have noticed this kind of error immediately.

**Comment by artoonl** [comment]
I understand that, thank you for your reply. 
FYI: I did not change any code (that I was aware of). 
When I get back to the version just before the upgrade (1.9.18 -> 1.10.24 all is working fine, after the upgrade it is wrong again. When I completely cleared  the site URL in the Atrocore settings and do a clear cache this functionality is working again! When I set is back its broken again.  I will test the application without the URL setting... Thnx for your help, and sorry for any novice questions from my side.
